### PR TITLE
Add getSuites() implementation

### PR DESCRIPTION
**Pull request by topikachu**
Adopt the upstream Junit plugin new getSuites() change.  jenkinsci/junit-plugin#464
Align the Jenkins version with Junit Plugin

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/junit/storage/database/DatabaseTestResultStorage.java [hunk 1]
```diff
@@ -25,12 +25,8 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Types;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```
Comment (timja) [inline src/main/java/io/jenkins/plugins/junit/storage/database/DatabaseTestResultStorage.java:28]: revert please and explicitly import

**Review by timja — COMMENTED** [review]
can you adapt the smokes test to add a test case for this please?

https://github.com/topikachu/junit-sql-storage-plugin/blob/eab796de4ad9c73c76e0b0c429aa55982d93889e/src/test/java/io/jenkins/plugins/junit/storage/database/DatabaseTestResultStorageTest.java#L112

**Comment by topikachu** [comment]
> can you adapt the smokes test to add a test case for this please?
> 
> https://github.com/topikachu/junit-sql-storage-plugin/blob/eab796de4ad9c73c76e0b0c429aa55982d93889e/src/test/java/io/jenkins/plugins/junit/storage/database/DatabaseTestResultStorageTest.java#L112

Done

**Comment by topikachu** [comment]
@timja Could you please take a look.at the latest change?
Thanks.

**Review by timja — APPROVED** [review]

**Pull request merged**